PROTO_tests/tests/string_test.py

```diff
@@ -414,21 +417,25 @@ def test_stick(self, size):
         delim = self.delim(np_base_words)
         self._stick_help(strings, test_strings, base_words, delim, size)
         self._stick_help(strings, test_strings, base_words, np.str_(delim), size)
-        self._stick_help(
-            strings, test_strings, base_words, str.encode(str(delim)), size
-        )
+        self._stick_help(strings, test_strings, base_words, str.encode(str(delim)), size)
 
         # Test gremlins delimiters
         g = self._get_ak_gremlins(size)
-        self._stick_help(g.gremlins_strings, g.gremlins_test_strings, g.gremlins_base_words, " ", size+3)
-        self._stick_help(g.gremlins_strings, g.gremlins_test_strings, g.gremlins_base_words, "", size+3)
-        self._stick_help(g.gremlins_strings, g.gremlins_test_strings, g.gremlins_base_words, '"', size+3)
+        self._stick_help(
+            g.gremlins_strings, g.gremlins_test_strings, g.gremlins_base_words, " ", size + 3
+        )
+        self._stick_help(
+            g.gremlins_strings, g.gremlins_test_strings, g.gremlins_base_words, "", size + 3
+        )
+        self._stick_help(
+            g.gremlins_strings, g.gremlins_test_strings, g.gremlins_base_words, '"', size + 3
+        )
 
     def test_str_output(self):
         strings = ak.array(["string {}".format(i) for i in range(0, 101)])
         assert (
-            "['string 0', 'string 1', 'string 2', ... , 'string 98', 'string 99', 'string 100']" ==
-            str(strings),
+            "['string 0', 'string 1', 'string 2', ... , 'string 98', "
+            "'string 99', 'string 100']" == str(strings)
         )
```

Comment:
same deal, i prefer declaring on a separate line to avoid the wrap around
```python
str_ans = "['string 0', 'string 1', 'string 2', ... , 'string 98', 'string 99', 'string 100']"
assert str_ans == str(strings)
```

Reply:
Updated